src/data/events.ts

```diff
@@ -142,7 +142,7 @@ export async function getNext3Events(): Promise<
   const today = new Date();
   today.setHours(0, 0, 0, 0);
   const request = {
-    sort: 'date',
+    sort: 'start',
```

Comment:
So this was a surprisingly easy fix :) . I think it's still from the time we had only one date and not a start and a end date.

Reply:
@Han2-Ro oh right. I wonder if we can somehow make sure this doesn't happen again? For example by using the payload types.

Reply:
yeah but I have no idea how you would take the name of a property (here: event.start) and use it as the value of a string. Or is there some other way?

Reply:
I think we should think about this another time as it's ok right now. I have some ideas we can talk about tomorrow like building a small client abstraction.